assets/upgradePatches.json

```diff
@@ -135,6 +135,18 @@
         "name": "kubevirt-plugin-service",
         "namespace": "kubevirt-hyperconverged"
       }
+    },
+    {
+      "semverRange": "<1.10.0",
+      "groupVersionKind": {
+        "group": "tektontasks.kubevirt.io",
+        "version": "apiregistration.k8s.io/v1",
+        "kind": "CustomResourceDefinition"
+      },
+      "objectKey": {
+        "name": "v1alpha1.tektontasks.kubevirt.io",
+        "namespace": "kubevirt-hyperconverged"
```

Comment:
CRD is cluster-scoped and doesn't have a namespace.
Please use an empty string

Reply:
Done.